src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -0,0 +1,70 @@
+package org.interledger.core;
```

Comment:
> we're also saying that SPSP is an "Interledger Protocol" or "ILP."

I'd disagree with this. SPSP is a setup protocol and a setup protocol is a prerequisite for making a payment using ILP. i.e. Before you can make an ILP payment you need to know what the ILP Address and amount are that you put in the packet and also the condition to attach to the first transfer.

It's somewhat analogous to DNS vs IP in my mind.